ZuluContent/Zulu/Spells/TriElemental/TriElementalSpell.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using System.Threading.Tasks;
+using Server;
+using Server.Engines.Magic;
+using Server.Spells;
+using Server.Targeting;
+
+namespace Scripts.Zulu.Spells.TriElemental
+{
+    public class TriElementalSpell : MagerySpell, ITargetableAsyncSpell<Mobile>
```

Comment:
This looks more accurate but I don't it really makes sense as a spell. Is the reason you've made it a spell so you can use the async methods? If so I think best of both worlds would be to keep TriElemental as a hitscript, then just implement the functionality in a `private static async void` method that gets called by OnHit.

e.g.

```csharp
public override void OnHit(Mobile attacker, Mobile defender, ref int damage)
{
    if (!Validate(attacker))
        return;

    Cast(attacker, defender);
}

private static async void Cast(Mobile caster, Mobile defender)
{
    // Copy paste code from OnTargetAsync here
}
```

Reply:
Yea that's a point, can change it